scripts/trilinos-integration/README.md

```diff
@@ -0,0 +1,59 @@
+![Kokkos](https://avatars2.githubusercontent.com/u/10199860?s=200&v=4)
+
+# Kokkos: Promotion Test Debugging
+
+This explains the use (and basic implementation details) of promotion testing
+new Kokkos and Kokkos Kernels branches. We first introduce a test for first
+validating an existing Trilinos branch that all tests should be passing on (usually the develop, master, or kokkos-promotion branch of Trilinos).
+After validating a clean promotion branch, we show how to test the same Trilinos branch
+with an updated Kokkoks and Kokkos Kernels.
+
+# Clean Promotion Test
+There is a script called `clean_promotion_test` in `scripts/trilinos-integration` that tests a self-contained
+Trilinos branch with its default included Kokkos and Kokkos Kernels packages.
+The script takes two arguments:
```

Comment:
It appears that the script assumes that you are in the [whatever]/kokkos/scripts/trilinos-integration directory, but that is not explicitly stated.  I think it might be better to not have such hard bindings in the scripts...The biggest issue / only thing I had to tweak calling it from somewhere else is the path to nvcc_wrapper.